infra/app/db.bicep

```diff
@@ -0,0 +1,24 @@
+param name string
+@description('Location for all resources.')
+param location string = resourceGroup().location
+param tags object = {}
+param keyValutName string
+param databaseName string
+param connectionStringKey string
+
+module sql '../core/database/sqlserver/sqlserver.bicep' = {
+  name: '${name}-sql-module'
+  params: {
+    name: name
+    location: location
+    databaseName: databaseName
+    keyVaultName: keyValutName
+    connectionStringKey: connectionStringKey
+    sqlAdminPassword: 'Pass@word'
+    appUserPassword: 'Pass@word'
```

Comment:
Please see the csharp-sql template for how we deal with passwords in templates and use that model.